src/solver/optimisation/opt_rename_problem.cpp

```diff
@@ -18,347 +18,309 @@ std::string BuildName(const std::string& name,
     return result;
 }
 
-void VariableNamer::SetLinkVariableName(unsigned int var, const std::string& variableType)
+void Namer::SetLinkElementName(unsigned int variable, const std::string& variableType)
 {
     const auto location = origin_ + AREA_SEP + destination_;
     targetUpdater_.UpdateTargetAtIndex(
       BuildName(variableType, LocationIdentifier(location, LINK), TimeIdentifier(timeStep_, HOUR)),
-      var);
+      variable);
 }
 
-void VariableNamer::SetAreaVariableName(unsigned int var, const std::string& variableType)
+void Namer::SetAreaElementName(unsigned int variable, const std::string& variableType)
 {
     targetUpdater_.UpdateTargetAtIndex(
       BuildName(variableType, LocationIdentifier(area_, AREA), TimeIdentifier(timeStep_, HOUR)),
-      var);
+      variable);
 }
 
-void VariableNamer::SetAreaVariableName(unsigned int var, const std::string& variableType, int layerIndex)
+void VariableNamer::SetAreaVariableName(unsigned int variable,
```

Comment:
```suggestion
void VariableNamer::SetAreaLayerVariableName(unsigned int variable,
```
Since this functions adds "Layer", it seems better to indicate it.